src/designer/elsa-workflows-designer/src/modules/notifications/toast-manager.tsx

```diff
@@ -0,0 +1,23 @@
+import { Component, Host, h } from '@stencil/core';
+import notificationStore from "./notification-store";
+
+@Component({
+  tag: 'elsa-toast-manager',
+  shadow: false,
+})
+
+export class ToastManager {
+
+  render() {
+    const {notifications} = notificationStore;
+    const notification = notifications.find(notification => notification.showToast !== false);
+
+    if (notification) {
+      return (
+          <elsa-awhile-notifications notification={notification}></elsa-awhile-notifications>
```

Comment:
I know you did not come up with this name, but can I ask you to rename "elsa-awhile-notifications" to something else? It's such a strange name 😄